libs/vscode/lsp-client/src/lib/nxls-client.ts

```diff
@@ -102,7 +102,7 @@ export class NxlsClient {
   }
 
   public async refreshWorkspace() {
-    window.withProgress(
+    await window.withProgress(
```

Comment:
Does this fix the spamming refresh issue?

Reply:
yes!